### DISCUSSION TITLE
Add CalendarDate field type

### DISCUSSION THREAD

**Discussion by ChuckJonas** [keystonejs/keystone#6118] [Feature requests]
Currently there is no good way to express a CalendarDate in keystone.  Yes, you could use timestamp, but sometimes less resolution is actually better.

Consider holidays.  xmas doesn't happen at one specific moment globally.  With timestamps, localization might cause xmas to display as the 24 or 26.

**Comment by chrisleyva** [comment]
Keystone used to have a `CalendarDay` field which is what you're suggesting.

I found the release where this field was removed: https://keystonejs.com/releases/2021-03-30

I'm not sure if it will come back in a future update, based on [this](https://keystonejs.com/docs/guides/keystone-5-vs-keystone-next#feature-parity) and [the roadmap](https://keystonejs.com/updates/roadmap) (under Field Types Review)

But I agree whole-heartedly. I think a field to express just a date would be a great addition!

**Comment by bladey** [comment]
Thanks @ChuckJonas and @chrisleyva, we'll take this into account!
